mackup/applications/fisherman.cfg

```diff
@@ -0,0 +1,6 @@
+[application]
+name = Fisherman
+
+[configuration_files]
+.config/fisherman
```

Comment:
@sallar @laughedelic Thanks for the mention. ~/.config/fisherman is like `node_modules`, so it should definitely _not_ be synced. The fishfile is similar to `package.json`'s `pkg.dependencies`.

If you have installed fisherman, running:

```
fisher
```

in your terminal will cause fisherman to install everything listed in the fishfile.